modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/drag-and-drop/getHandlerDrop.es.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {
+	dropCustomObjectField,
+	dropFieldSet,
+	dropLayoutBuilderField,
+} from '../actions.es';
+import {
+	DRAG_DATA_DEFINITION_FIELD,
+	DRAG_FIELDSET,
+	DRAG_FIELD_TYPE,
+} from './dragTypes.es';
+
+export const getHandlerDrop = ({dataDefinition, dataLayoutBuilder}) => {
```

Comment:
@matuzalemsteles this name sounds odd. Maybe `getDropHandler` is what you were looking for? Or `createDropReducer` or `makeDropHandler`?

Reply:
Yeah, sure 😅. I think that just `getDropHandler` would be enough, I go with that. Thanks